workspaces/Cargo.toml

```
@@ -26,15 +26,15 @@ tokio-retry = "0.3"
 tracing = "0.1"
 url = { version = "2.2.2", features = ["serde"] }
 
-near-account-id = "0.5"
-near-crypto = "0.5"
-near-primitives = "0.5"
-near-jsonrpc-primitives = "0.5"
-near-jsonrpc-client = { version = "0.1", features = ["sandbox"] }
-near-sandbox-utils = "0.1.1"
+near-account-id = "0.12"
+near-crypto = "0.12"
+near-primitives = "0.12"
+near-jsonrpc-primitives = "0.12"
+near-jsonrpc-client = { git = "https://github.com/near/near-jsonrpc-client-rs", features = ["sandbox"] }
```

`near-jsonrpc-client` `v0.3.0` has been released. https://github.com/near/near-jsonrpc-client-rs/releases/tag/v0.3.0


```suggestion
near-jsonrpc-client = { version = "0.3", features = ["sandbox"] }
```